build/charts/antrea/conf/antrea-agent.conf

```
@@ -306,10 +306,10 @@ kubeAPIServerOverride: {{ .Values.kubeAPIServerOverride | quote }}
 # 10.96.0.10:53, [fd00:10:96::a]:53).
 dnsServerOverride: {{ .Values.dnsServerOverride | quote }}
 
-# The minTTL setting helps address the problem of applications caching DNS response IPs beyond the TTL value for the DNS record.
+# The fqdnCacheMinTTL setting helps address the problem of applications caching DNS response IPs beyond the TTL value for the DNS record.
 # It is used to enforce FQDN policy rules, ensuring that resolved IPs are included in datapath rules for as long as the application is caching them.
 # This value should ideally be set to the maximum caching duration across all applications.
-minTTL: {{ .Values.minTTL }}
+fqdnCacheMinTTL: {{ .Values.minTTL }}
```

```suggestion
fqdnCacheMinTTL: {{ .Values.fqdnCacheMinTTL }}
```

This is why the manifests are not generated correctly (`fqdnCacheMinTTL: ` instead of `fqdnCacheMinTTL: 0`)

---

Sorry , my bad. Will correct that.